### PR TITLE
MLPAB-2029 - create an account level opensearch serverless collection

### PR DESCRIPTION
**Pull request by andrewpearce-digital**
# Purpose

Share a single collection to reduce costs

Fixes MLPAB-2029

## Approach

- Create an account level collection

## Learning

Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the Modernising LPA service

### CODE REVIEW

File: terraform/account/opensearch.tf [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+      ],
+      Principal = [
+        "arn:aws:iam::${data.aws_caller_identity.global.account_id}:role/operator"
+      ]
+    }
+  ])
+  provider = aws.eu_west_1
+}
+
+resource "aws_opensearchserverless_access_policy" "team_breakglass_access" {
```
Comment (acsauk) [inline terraform/account/opensearch.tf:91]: Would we always want this available on production? Or have we decided opensearch is always off-limits for live data?

Comment (acsauk) [reply terraform/account/opensearch.tf:91]: Oh ignore me, I've just seen the ordering is different here for `count` 🙄

File: terraform/account/terraform.tfvars.json [hunk 2]
```diff
@@ -6,23 +6,26 @@
...
     },
     "production": {
       "account_id": "313879017102",
       "account_name": "production",
       "is_production": true,
       "regions": [
         "eu-west-1"
-      ]
+      ],
+      "pagerduty_service_name": "OPG Modernising LPA Non-Production"
```
Comment (acsauk) [inline terraform/account/terraform.tfvars.json:28]: Should this be:

```suggestion
      "pagerduty_service_name": "OPG Modernising LPA Production"
```

Comment (andrewpearce-digital) [reply terraform/account/terraform.tfvars.json:28]: We don't currently have a production pagerduty service to alert to, so this isn't a desired name but the name of an existing resource

File: terraform/account/opensearch.tf (hunk 1 above)
Comment (andrewpearce-digital) [reply terraform/account/opensearch.tf:91]: Breakglass is an alarmed role (using it raises an alarm) for use in emergencies.

**Pull request merged**